pulpcore/cli/core/repository.py

```diff
@@ -1,20 +1,35 @@
+from typing import Iterable
```

Comment:
```suggestion
import typing as t
```
Seems to be more convenient in the long run.